test/utils/utils.go

```diff
@@ -167,7 +167,7 @@ func (p *PodmanTest) NumberOfPods() int {
 // This function assumes only one container is active.
 func (p *PodmanTest) GetContainerStatus() string {
 	var podmanArgs = []string{"ps"}
-	podmanArgs = append(podmanArgs, "--all", "--format={{.Status}}")
+	podmanArgs = append(podmanArgs, "--all", "--format={{.State}}")
```

Comment:
This actually seems pretty significant - is this a breaking change from Docker?

Reply:
yes, it is a breaking change with Docker

Reply:
I think we need to either alias `Status` to `State` or change it back completely, then

Reply:
We could rename:
https://github.com/containers/libpod/blob/master/cmd/podman/containers/ps.go#L268 to Status() or add a Status() method that calls State().  /cc @baude Was State() chosen for a reason?

Reply:
thanks for the hint, I will add a function Status() that calls State() for now.